src/__tests__/rules.test.js

```diff
@@ -2,7 +2,14 @@ import { existsSync } from 'fs';
 import { resolve } from 'path';
 import { rules } from '../';
 
-const ruleNames = Object.keys(rules);
+const ruleNames = Object.keys(rules).filter(rule => {
+  const excludeRules = [
```

Comment:
can you create this array outside of the `filter`?

Also maybe a separate test that all "excluded" rules have the `meta.deprecated` property set?

Reply:
> Also maybe a separate test that all "excluded" rules have the meta.deprecated property set?

I thought about that, but I think that is only wanted for renamed rules. If a rule is deprecated without renaming, you'll probably want to keep it inside this test. Right?

Reply:
We use `import` & `export` 🙂 And yeah, I landed the Babel change to make the TS migration have a smaller diff

Reply:
> We use import & export

Ha yeah it should have been obviously. Just wasn't thinking. Silly me.

Reply:
good point!